src/app/@ansyn/ansyn/modules/plugins/openlayers/plugins/visualizers/tools/measure-distance.visualizer.ts

```diff
@@ -342,6 +342,7 @@ export class MeasureDistanceVisualizer extends EntitiesVisualizer {
 
 	private setLabelsFeature() {
 		if (!this.measureData.meausres.length) {
+			// TODO: Do we want to get data from the server here? if so, use refresh()
```

Comment:
we wanted to clear the source vector (when no items were received) not to refresh it